pkgs/applications/science/math/gfm/default.nix

```diff
@@ -0,0 +1,54 @@
+{ stdenv
+, fetchurl
+, fetchpatch
+, pkgconfig
+, autoreconfHook
+, gtk2
+, gnome2
+, glib
+, libtifiles2
+, libticables2
+, libticalcs2
+, libticonv
+}:
+
+stdenv.mkDerivation rec {
+  pname = "gfm";
+  version = "1.08";
+  src = fetchurl {
+    url = "mirror://sourceforge/tilp/${pname}-${version}.tar.bz2";
+    sha256 = "0zq1a9mm54zr18dz2mqh79w1a126xwqz6dcrpjlbd1pnmg01l0q9";
+  };
+
+  patches = fetchpatch {
+    name = "remove-broken-kde-support.patch";
+    url = "https://aur.archlinux.org/cgit/aur.git/plain/remove-broken-kde-support.patch?h=gfm";
+    sha256 = "03yc8s2avicmv04f2ygg3r3q8l7kpsc94mhp6clp584kmjpjqag5";
+  };
+
+  nativeBuildInputs = [
+    autoreconfHook
+    pkgconfig
+  ];
+
+  buildInputs = [
+    gnome2.gtk
```

Comment:
gtk2 or gtk3?

Reply:
does this possibly need wrapGAppsHook (does this have an UI and does that ui work?)

Reply:
It needs gtk2: `checking for gtk+-2.0 >= 2.6.0`.
It has an UI, and it does work, so I guess it doesn't need wrapGAppsHook

Reply:
Is gtk2 and gnome2.gtk the same packages?

Reply:
Yes, they are the same package.
In `pkgs/desktops/gnome-2/default.nix`:
```nix
  gtk = pkgs.gtk2;
```

Reply:
Then I would prefer using gtk2.